apps/ui/src/components/Modal/Account.vue

```diff
@@ -1,24 +1,43 @@
-<script setup>
+<script setup lang="ts">
 import { getInjected } from '@snapshot-labs/lock/src/utils';
 import { shorten, explorerUrl } from '@/helpers/utils';
 import connectors, { mapConnectorId, getConnectorIconUrl } from '@/helpers/connectors';
 
+const win = window;
```

Comment:
Don't think this is needed anymore

Reply:
Without it it complains about types, not sure if it's some Vue magic hack, otherwise I think we would need to extend Window interface.